editor/src/clj/editor/keymap.clj

```diff
@@ -700,14 +701,20 @@
 (defn install!
   "Install a keymap on a scene, replacing any predefined accelerators"
   [keymap ^Scene scene execute-fn]
-  (doto (.getAccelerators scene)
```

Comment:
The problem was that scene accelerators are mutated both by us here and by JavaFX on Windows and Linux when using a menubar with accelerators. Using `KEY_PRESSED` handler instead helps.